docs_source_files/content/grid/grid_4/graphql_support.de.md

```diff
@@ -44,33 +44,59 @@ The structure of grid schema is as follows:
     grid: {
         uri,
         totalSlots,
-        usedSlots,
+        nodeCount,
+        maxSession,
         sessionCount,
-        nodes : [
-            {
+        version,
+        sessionQueueSize
+    }
+    sessionsInfo: {
+        sessionQueueRequests,
+        sessions: {
+            id,
+            capabilities,
+            startTime,
+            uri,
+            nodeId,
+            nodeUri,
+            sessionDurationMillis
+            slot : {
+                id,
+                stereotype,
+                lastStarted
+            }
+        }
+    }
+    nodesInfo: {
+        nodes {
```

Comment:
I should probably move this back to an array. Wasn't sure if this schema example was in the syntax of how you make queries.

Will make an update

Reply:
Yes, it should be an array.